torch/onnx/symbolic_opset9.py

```diff
@@ -395,14 +401,15 @@ def addcmul(g, self, tensor1, tensor2, value=1.0):
 def _div_rounding_mode(g, self, other, rounding_mode):
     if rounding_mode is None:
         return true_divide(g, self, other)
-    elif rounding_mode == "floor":
+    if rounding_mode == "floor":
```

Comment:
What is the reasoning for this? `elif` can give more context to JIT compilers optimize things out as opposed to a generic `if`

Reply:
Applying early returns and so there's no need for elif or else here. Thoughts?

Reply:
In general, `elif` can be faster than `if` because sequential `if`s are checked one after the other; while a `elif` chain can be bypassed when the first condition is matched.

Even with returns inside each `if`, the JIT compiler can waste time parallelizing each `if` in a different core that could deprioritized otherwise. 

Besides from the performance point of view, `elif` _explicitly_ states the mutual exclusivity of the conditions being evaluated, which IMO is a better business documentation than disconnected `if`s

Reply:
Makes sense. I will revert the change

Reply:
Done